pkg/catalog/prepare_test.go

```diff
@@ -0,0 +1,215 @@
+package catalog_test
+
+import (
+	"bytes"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/weaveworks/pctl/pkg/catalog"
+	gitfakes "github.com/weaveworks/pctl/pkg/git/fakes"
+)
+
+type mockTransport struct {
+	res *http.Response
+}
+
+func (t *mockTransport) RoundTrip(req *http.Request) (*http.Response, error) {
+	return t.res, nil
+}
+
+var _ = Describe("prepare", func() {
+	It("can prepare the environment with everything that profiles needs with dry run", func() {
```

Comment:
can this `It` be a little clearer? like "when dry run is set, prepares all the manifests that profiles requires but does not apply them to the cluster" or something

Reply:
@Callisto13 Is it better now? I wrapped it into a When to make the context more clearer.

Reply:
yep way nicer to read, ty!